CONTRIBUTING.adoc

```diff
@@ -5,225 +5,147 @@ OpenShift Developers <dev@lists.openshift.redhat.com>
-
-        $ oadm registry
-
-
-10.  At this point it may be helpful to load some image streams and templates.  These commands will make use of fixtures from the `openshift/origin/examples` dir:
-
-        # load image streams
-        $ oc create -f /data/src/github.com/openshift/origin/examples/image-streams/image-streams-centos7.json -n openshift
-        # load templates
-        $ oc create -f /data/src/github.com/openshift/origin/examples/sample-app/application-template-stibuild.json -n openshift
-        $ oc create -f /data/src/github.com/openshift/origin/examples/db-templates -n openshift
-
-
-11. At this point you can open a browser on your host system and navigate to https://localhost:8443/console to view the web console.  You can log in with any username and password combination.
-
-
-12.  NOTE: to properly stop OpenShift and clean up, so that you can start fresh instance of OpenShift, execute:
-
-    # shut down openshift
-    $ sudo pkill openshift
-    # stop the docker containers
-    $ docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
-    # deleting all the internal config files, etcd, etc and starting openshift fresh
-    $ sudo rm -rf openshift.local.*
-    # if you used the --volume-dir=/home/vagrant/volumes flag, then run these
-
-TIP: See https://github.com/openshift/vagrant-openshift for more advanced options
-
-==== Ensure virtual box interfaces are not managed by Network Manager
-
-If you are developing on a Linux host, then you need to ensure that Network Manager is ignoring the
-virtual box interfaces, otherwise they cause issues with multi-vm networking.
-
-Follow these steps to ensure that virtual box interfaces are unmanaged:
-
-1. Check the status of Network Manager devices:
-
-   $ nmcli d
-
-2. If any devices whose name start with vboxnet* are not unmanaged, then they need to be added to
-   NetworkManager configuration to be ignored.
-
-   $ cat /etc/NetworkManager/NetworkManager.conf
-
-        [keyfile]
-        unmanaged-devices=mac:0a:00:27:00:00:00;mac:0a:00:27:00:00:01;mac:0a:00:27:00:00:02
-
-3. One can use the following command to help generate the configuration:
-
-   $ ip link list | grep vboxnet  -A 1 | grep link/ether | awk '{print "mac:" $2}' |  paste -sd ";" -
-
-4. Reload the Network Manager configuration:
-
-    $ sudo nmcli con reload
+3. Open up a new terminal or source the changes in your current terminal.  Then
+   clone your forked repo:+
+----
+$ mkdir -p $GOPATH/src/github.com/openshift
+$ cd $GOPATH/src/github.com/openshift
+$ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+$ cd origin
+$ git remote add upstream git://github.com/openshift/origin
+----
+4.  From here, you can generate the OpenShift binaries and images by running:+
+----
+$ make release
+----
+    If you want to generate just the OpenShift binaries, you could run:+
+----
+$ make build
```

Comment:
since you assume they are going to run their changes via cluster up later, this isn't all that helpful other than making sure it compiles cleanly before invoking make release.